python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -102,8 +102,18 @@ class RandomForestClassifier(BaseRandomForestModel, DelayedPredictionMixin,
         per node split.
     n_bins : int (default = 8)
         Number of bins used by the split algorithm.
-    min_rows_per_node : int (default = 2)
```

Comment:
We should deprecate min_rows_per_node. It's not a very commonly-used parameter, but it has been out there. We can keep the argument but change it default to None and issue a deprecation warning with a note about the new params if it is passed in as not None.

Reply:
Got it. We can issue a deprecation warning.

Reply:
Added a deprecation warning.